frontend/components/receipts/utils/order/build-order-columns.tsx

```diff
@@ -8,6 +9,21 @@ import { OrderMarketComponent } from './order-market'
 import { OrderTypeComponent } from './order-type'
 import { PeggedOrderInfo } from './pegged-order-info'
 import { Direction } from './direction'
+import { formatDateWithLocalTimezone } from '@vegaprotocol/utils'
+import { nanoSecondsToMilliseconds } from '../../../../lib/utils.ts'
+import { CopyWithCheckmark } from '../../../copy-with-check'
+
+const orderStatuses = {
```

Comment:
Let's ensure typechecking of all keys:

```suggestion
const orderStatuses: Record<vegaOrderStatus, string> = {
```